packages/opentelemetry-metrics/src/SumObserverMetric.ts

```diff
@@ -49,15 +48,11 @@ export class SumObserverMetric
     observerResult.values.forEach((value, labels) => {
       const instrument = this.bind(labels);
       // SumObserver is monotonic which means it should only accept values
-      // greater or equal then previous value
-      const previous = instrument.getAggregator().toPoint();
-      let previousValue = -Infinity;
-      if (previous.timestamp[0] !== 0 || previous.timestamp[1] !== 0) {
-        previousValue = previous.value as LastValue;
-      }
-      if (value >= previousValue) {
-        instrument.update(value);
-      }
+      // greater or equal then previous value.
+      // However it is impractical for the stateless accumulator to remember
```

Comment:
As the aggregator in the Accumulator (class `Metric` in the context) no longer bookkeeping all the records of past label-aggregator pairs, it is impractical for SumObserver to determine if it is updating with a rising value (unless the observer itself memos last value, but I'm not favoring the workaround). Some practices may recognize the value drop as a reset and correctly reflect the true trend, So I'd believe the value drop in SumObserver can reflect the true trends rather than discarding data frames.

Reply:
SumObserver cannot have value drop, this is monotonic metric - it can only raise or remain the same.

Reply:
Yeah, you are right. I've checked again the status quo PR of the processor specs, SumObserver and
UpDownSumObserver instruments are configured for Cumulative aggregation temporality in `StatelessExportKindSelector`.